examples/sycl/pvc/pvc_gemm.cpp

```diff
@@ -63,7 +63,7 @@ struct Options {
   Options():
     help(false),
     error(false),
-    m(5120), n(4096), k(4096), l(1), iterations(20),
+    m(512), n(512), k(16), l(1), iterations(1),
```

Comment:
why is this marked as resolved? Even if we want to make this change to the default should this really be part of this PR. For both review and resolving the accuracy question it would help if the PR would be broken into multiple independent smaller ones.